projects/go-lib/src/lib/components/go-toaster/go-toaster.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface ToastInterface {
+  header?: string;
+  message?: string;
+  type?: string;
+}
```

Comment:
Should any of these be required?

Reply:
Thought about that, this interface is only used in the service where the type and message are always set. Because those methods take a parameter of type ToastInterface it would require the user to pass these when calling the method.